### DISCUSSION TITLE
Documentation: comparision with using EDK2

### DISCUSSION THREAD

**Discussion by tobiasBora** [nix-community/raspberry-pi-nix#65] [General]
This project looks great, but I'm quite curious to understand: the wiki mentions that it is better to use EDK2 + the rpi5 kernel that is now in nixpkgs: https://github.com/NixOS/nixos-hardware/pull/927

How does both approaches compare, e.g. in term of caching/hardware support? A note in the readme would be great to clarify this.

**Comment by tstat** [comment]
I don't know anything about EDK2, so that is something for me to look into. A short partial answer is that this repo uses the bootloader process that raspberry pi OS itself uses, and the rpi 5 kernel should be the same (raspberry pi's linux fork), although pointing to different pins I'm sure.
